core/src/test/java/org/elasticsearch/index/query/BaseQueryTestCase.java

```diff
@@ -322,4 +315,21 @@ protected static void assertValidate(QueryBuilder queryBuilder, int totalExpecte
             assertThat(queryValidationException, nullValue());
         }
     }
+
+    /**
+     * Helper method to return a random rewrite method
+     */
+    protected static String getRandomRewriteMethod() {
```

Comment:
this is in the base class but it's used only in the prefix tests. will it be used in other tests in the future?